README.md

```diff
@@ -218,14 +218,15 @@ You can experiment with the class fields proposal using the following complete i
 - Public fields are [enabled by default](https://www.chromestatus.com/feature/6001727933251584) in Chrome 72 / V8 7.2
 - Private fields are [enabled by default](https://www.chromestatus.com/feature/6035156464828416) in Chrome 74 / V8 7.4
 - Public instance fields are [enabled by default](https://bugzilla.mozilla.org/show_bug.cgi?id=1499448) in Firefox 69
+- Public static fields are [enabled](https://bugzilla.mozilla.org/show_bug.cgi?id=1535804) in Firefox 75
```

Comment:
I think the way to phrase this is, it's enabled in Firefox Nightly as of (date).